baseplate/lib/prometheus_metrics.py

```diff
@@ -343,3 +343,60 @@ def getHTTPSuccessLabel(httpStatusCode: int) -> str:
     The HTTP success label is "true" if the status code is 2xx or 3xx, "false" otherwise.
     """
     return str(200 <= httpStatusCode < 400).lower()
+
```

Comment:
🔕  extra line returns?

Reply:
I could keep that function at the end of the file if you think that makes more sense though